### ISSUE TITLE
[bug] TypeError when editing fields in ACF 6.0.2 

### ISSUE THREAD

**Issue by joshuafredrickson** [Log1x/acf-phone-number#51]
It looks like there's a conflict with ACF 6.0. With the plugin activated, I get the following error when editing fields, which results in general bugginess. 

```
TypeError: Cannot read properties of undefined (reading 'getAttribute')
    at new i (https://example.test/wp-content/plugins/acf-phone-number/public/js/field.js:1:10819)
    at https://example.test/wp-content/plugins/acf-phone-number/public/js/field.js:1:34487
    at HTMLDivElement.<anonymous> (https://example.test/wp-content/plugins/acf-phone-number/public/js/field.js:1:285)
    at Function.each (https://example.test/wp-admin/load-scripts.php?c=0&load%5Bchunk_0%5D=jquery-core,jquery-migrate,utils,regenerator-runtime,wp-polyfill,wp-hooks&ver=6.0.2:2:3003)
    at s.fn.init.each (https://example.test/wp-admin/load-scripts.php?c=0&load%5Bchunk_0%5D=jquery-core,jquery-migrate,utils,regenerator-runtime,wp-polyfill,wp-hooks&ver=6.0.2:2:1481)
    at https://example.test/wp-content/plugins/acf-phone-number/public/js/field.js:1:184
    at https://example.test/wp-content/plugins/advanced-custom-fields-pro/assets/build/js/acf-input.min.js?ver=6.0.2:1:2262
    at o (https://example.test/wp-content/plugins/advanced-custom-fields-pro/assets/build/js/acf.min.js?ver=6.0.2:1:1417)
    at Object.doAction (https://example.test/wp-content/plugins/advanced-custom-fields-pro/assets/build/js/acf.min.js?ver=6.0.2:1:587)
    at n.doAction (https://example.test/wp-content/plugins/advanced-custom-fields-pro/assets/build/js/acf.min.js?ver=6.0.2:1:18166) 
```

This appears to be isolated to the edit screen... Things seem to work fine when actually filling the field.

WP 6.0.2
ACF 6.0.2
ACF Phone Number 1.1.9

**Comment by huubl** [comment]
I'm experiencing the same issue here. It breaks ACF fields editing screen.

**Comment by Log1x** [comment]
Not sure why this is suddenly broken but I can see why. That aside, it seems that even if I comment out https://github.com/Log1x/acf-phone-number/blob/master/src/PhoneNumberField.php#L109-L110 the Select2 field for setting the default country in the editor screen works, but looks broken.

Is this the same for you?

**Comment by Log1x** [comment]
Could you both try
```
$ composer require log1x/acf-phone-number:dev-fix/check-phone-country-exists
```
and see if it fixes your issues outside of my screenshot above?

**Comment by joshuafredrickson** [comment]
That fixed it for me! Works in both ACF 5.12.x and 6.0.2.
